cpg-core/src/main/java/de/fraunhofer/aisec/cpg/graph/Extensions.kt

```diff
@@ -68,12 +68,26 @@ inline fun <reified T : Node> Node.dfgFrom(): List<T> {
     return this.prevDFG.toList().filterIsInstance<T>()
 }
 
+fun matchesName(name: Name?, lookup: String): Boolean {
```

Comment:
isn't this very similar to the Name.endsWith function?

Reply:
Or should this be the equal method of Name?

Reply:
I think it should be similar to `Name.endsWith` and we can probably replace it